druid/src/widget/label.rs

```diff
@@ -477,6 +477,7 @@ impl<T: Data> Widget<T> for Label<T> {
         self.text_should_be_updated = false;
         if data_changed {
             let new_text = self.text.display_text();
+            dbg!(&new_text);
```

Comment:
I don't think this `dbg!` is intentional?

Reply:
😅